Src/WindowsServer/WindowsServer.Shared/FirstChanceExceptionStatisticsTelemetryModule.cs

```diff
@@ -77,14 +98,14 @@ public void Initialize(TelemetryConfiguration configuration)
                 {
                     if (!this.isInitialized)
                     {
-                        this.isInitialized = true;
-
                         this.telemetryClient = new TelemetryClient(configuration);
                         this.telemetryClient.Context.GetInternalContext().SdkVersion = SdkVersionUtils.GetSdkVersion("exstat:");
 
                         this.metricManager = new MetricManager(this.telemetryClient);
 
                         this.registerAction(this.CalculateStatistics);
+
+                        this.isInitialized = true;
```

Comment:
Won't moving this down increase the risk of re-entry by another thread (not that it locks so it's racing either way)?